### PR TITLE
Personal portfolio setup

### PR DESCRIPTION
**Pull request by 3nityGitHub**
# In this PR,

- I added the index.html file
- I added style.css file
- I installed linter.yml
- I installed stylelintrc.json
- I installed hintrc file
- I updated README.md file
- I Modified Html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            </li>
+            <li class="social_media_link">
+              <a href="https://www.angellist.com/"
+                ><i class="colored fa-solid fa-hand-peace fa-xl"></i
+              ></a>
+            </li>
+          </ul>
+        </section>
+      </div>
+    </div>
```
Comment (levy002) [inline index.html:75]: - [ ] Kindly wrap this headline container into a `section` tag. Generally, the `section` represents a generic standalone section of the document, so kindly use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.first_section_div {
+  height: auto;
+  width: 347px;
+}
```
Comment (levy002) [inline styles.css:9]: - [ ] Kindly make sure to adjust your `width`on this container in order to meet the provided design. :width:

    - ### Your design
 
     image

    - ### Figma design
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @3nityGitHub   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (3nityGitHub) [reply index.html:75]: @levy002,
Thanks for your review. I will effect the changes asap. But I need clarity, when you say headline container, are you referring to lines 25-40?

**Review by levy002 — APPROVED** [review]
### Hello @3nityGitHub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**